postgres/osmap.yaml

```diff
@@ -19,6 +19,17 @@ Debian:
   pkg_dev: postgresql-server-dev-all
   pkg_libpq_dev: libpq-dev
 
+  pkg: 'postgresql-{{ repo.version }}'
+  pkg_client: 'postgresql-client-{{ repo.version }}'
```

Comment:
Agree - The dilemma arises because they are defined in pillar.example for Debian, and osmap.yaml for RedHat and I don't like the inconsistency.  Open to suggestions including fixing in this PR or raising as issue and fixing in future PR.